analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/ObjectsShouldNotBeDisposedMoreThanOnce.cs

```diff
@@ -1,15 +1,30 @@
 ﻿using System;
 using System.IO;
+using System.Data.Common;
 
 public interface IInterface1 : IDisposable { }
```

Comment:
Let's rename this. And also all usages to avoid `interface1` below
```suggestion
public interface IInheritsDispose : IDisposable { }
```
or
```suggestion
public interface IHasDispose : IDisposable { }
```
or
```suggestion
public interface IWithDispose : IDisposable { }
```